### ISSUE TITLE
python build for windows and python 3.7

### ISSUE THREAD

**Issue by padmalcom** [alphacep/vosk-api#200]
Hi, first of all: thank you for your fantastic library!
I'm using vosk on windows 10 with python 3.8. Unfortunately, there are a lot of common libraries that do not work with python 3.8 yet, such as numpy, scikit-learn ...
It would be fantastic to build the whl files for the current vosk release for windows and python 3.7 or provide a (quick and dirty) checklist how to do it myself. I would even help to do the build on a regular basis.
Cheers

**Comment by nshmyrev** [comment]
Yeah, it would be nice to automate it actually with github actions and build for many versions at once.

**Comment by padmalcom** [comment]
That would be great. Do you have any information on how to build the whl files?

**Comment by nshmyrev** [comment]
As usual, `python3 setup.py bdist_wheel`, you just need to arrange kaldi build files.

**Comment by padmalcom** [comment]
`python3 setup.py bdist_wheel` fails even after I succeeded building kaldi. Is there a setup.py especially for windows? I was following your documentation here https://alphacephei.com/vosk/install.

Error message is the following (translated, so may differ to original message):
```
LINK : fatal error LNK1181: Can not open input file "C:\Users\admin\kaldi\src\online2\kaldi-online2.a" 
error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\Community\\VC\\Tools\\MSVC\\14.16.27023\\bin\\HostX86\\x64\\link.exe' failed with exit status 1181
```

Since setup.py is looking for a *.a file it seems that setup.py is meant to be used with linux. My folder in windows contains *.h and *.cc files only.

**Comment by nshmyrev** [comment]
On windows you link to .lib files instead of .a

**Comment by padmalcom** [comment]
Yes and I'd like to tell my setup.py script to do so but I don't know how :)

**Comment by nshmyrev** [comment]
Edit extension here https://github.com/alphacep/vosk-api/blob/dff4ab26e42e80c77ad742e179708e4c453d9a4b/python/setup.py#L32

**Comment by padmalcom** [comment]
I could solve the extension issue and linking all libraries, except gfortran. I'm not sure where the script is looking for the file and where to get it. Can you give me a hint?

**Comment by nshmyrev** [comment]
New version is released for Windows, should work with python 3.7

https://pypi.org/project/vosk/0.3.18/

please update, it should be better. Reopen if you see any issues.
